portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/security/ldap/LDAP.testcase

```diff
@@ -48,6 +48,64 @@ definition {
 		}
 	}
 
+	@description = "This is a use case for LPS-188257. Verify user is able to edit user info after enabling Use LDAP Password Policy."
+	@priority = 4
+	test CanEditUserAfterEnablingUseLDAPPasswordPolicy {
+		property apacheds.enabled = "true";
+		property test.liferay.virtual.instance = "false";
+		property test.run.type = "single";
+
+		task ("Given: Go to enable Use LDAP Password Policy") {
+			PortalSettings.gotoConfiguration(
+				configurationCategory = "LDAP",
+				configurationName = "General",
+				configurationScope = "Virtual Instance Scope");
+
+			FormFields.enableCheckbox(fieldName = "InstanceSettingsPortlet_passwordPolicyEnabled");
+
+			PortletEntry.save();
+		}
+
+		task ("And: Add LDAP server and configure settings") {
+			var enableSettingList = "enabled,ldapImportEnabled,ldapExportEnabled";
+
+			LDAP.addLDAPServerAndTestConnection();
+
+			LDAP.enableLDAPSettings(enableSettingList = ${enableSettingList});
+		}
+
+		task ("When: Add a new user") {
+			JSONUser.addUser(
+				userEmailAddress = "usera@liferay.com",
+				userFirstName = "firstName",
+				userLastName = "lastName",
+				userScreenName = "usera");
+
+			JSONUser.setFirstPassword(
+				agreeToTermsAndAnswerReminderQuery = "true",
+				requireReset = "false",
+				userEmailAddress = "usera@liferay.com");
+		}
+
+		task ("Then: Assert that user is able to edit user info") {
+			User.openUsersAdmin();
+
+			UserNavigator.gotoUser(userScreenName = "usera");
+
+			User.editUserInformation(
+				userEmailAddressEdit = "testadmin@liferay.com",
+				userFirstNameEdit = "testfirstName",
+				userLastNameEdit = "testlastName",
+				userScreenNameEdit = "testscreen");
+
+			User.viewUserInformation(
+				userEmailAddress = "testadmin@liferay.com",
+				userFirstName = "testfirstName",
+				userLastName = "testlastName",
+				userScreenName = "testscreen");
+		}
```

Comment:
Hi @JessieZhao0714, I think we need an additional logout and re-login step just to confirm, these data was saved to LDAP server also, so when you log back in, the data remains the same